app/templates/pains_verify_and_coefficient_use.html

```diff
@@ -64,7 +64,12 @@ <h3>Verify Pains Filter</h3>
             document.write('Display Color: <input type="color" name="mpo0Color"/><br>');
           }
         </script>
-      Tanimoto Coefficient: 
+        Fingerprint Type: 
+        <select id="fp_type" name="fp_type">
+          <option value="morgan" selected="selected">Morgan</option>
+          <option value="atom-pair">Atom Pair</option>
+        </select><br/>
+        Tanimoto Coefficient: 
```

Comment:
I think this could be really cool if we could find a way to lead people to the documentation about Morgan and atom pair fingerprints.